src/ray/core_worker/actor_manager.h

```diff
@@ -172,6 +180,14 @@ class ActorManager {
   void HandleActorStateNotification(const ActorID &actor_id,
                                     const rpc::ActorTableData &actor_data);
 
+  /// Function that's invoked when the actor is out of scope.
+  ///
+  /// \param actor_handle The actor handle that will be marked as invalidate.
+  void MakeActorInvalid(std::shared_ptr<ActorHandle> actor_handle);
+
+  /// Check if actor is valid.
+  bool IsValidActor(const ActorID &actor_id) const;
```

Comment:
Not really sure about the word "valid", can we make the name more descriptive? For example, does it just mean that we are subscribed to it or does it mean that we can submit tasks to it?

Reply:
> Not really sure about the word "valid", can we make the name more descriptive? For example, does it just mean that we are subscribed to it or does it mean that we can submit tasks to it?

How about renaming to `IsActorKilledOrOutOfScope` ?